src/common/paddle.ts

```diff
@@ -54,3 +63,125 @@ export const getPriceFromPaddleItem = (
 
   return priceAmount / 100;
 };
+
+export enum ProductPurchaseType {
+  Plus = 'plus',
+  Core = 'core',
+}
+
+export const getProductPurchaseType = ({
+  id,
+}: {
+  id: string;
+}): ProductPurchaseType => {
+  if (!remoteConfig.vars.coreProductId) {
```

Comment:
Learning from the mistakes of Plus, is it possible to use Paddle API somehow to detect if it was a core purchase?
Maintaining product ids on GB is something we often forget to do which breaks the flow, especially with experimentation. So ideally if we can have something more bullet proof it would be great

Reply:
Yep, good think paddle prefixes them so at least you can see that you passed a wrong one eg. `pro_`, `pri_`..